Crashteroids Starter/Assets/RW/Scripts/Laser.cs

```diff
@@ -50,10 +56,24 @@ private void OnCollisionEnter(Collision collision)
     {
         if (collision.gameObject.GetComponent<Asteroid>() != null)
         {
-            spawner.SpawnShrapnel(collision.gameObject.transform.position, collision.collider);
-            Game.AsteroidDestroyed();
+           if(collision.gameObject.GetComponent<Asteroid>().health < 1)
```

Comment:
Surely if an asteroid has zero health it would already be destroyed?

Reply:
This is required to ensure asteroid is destroyed when its health falls below 1.